cspell-custom-words.txt

```diff
@@ -0,0 +1,194 @@
+Acala
+acala
+Acks
+Alertmanager
+algod
+Algorand
+algosdk
+alist
+Aptos
+authorisation
+authorise
+authorised
+backdoors
+Bech
+bech
+behaviour
+bigset
+Bigtable
+borsh
+bscscan
+BUILDKIT
+bytecodes
+callstack
+CCTP
+Celestia
+celestia
+certusone
+Chainlink
+Coinspect
+collateralization
+colour
+Concat
+conftest
+Cosm
+cosmoshub
+Cosmwasm
+cosmwasm
+counterparty
+cpus
+crosschain
+Cyfrin
+datagram
+denoms
+devnet
+Dymension
+dymension
+EMMITTER
+ethcrypto
+ethersproject
+ETHRPC
+evmos
+forgeable
+FQTs
+frontends
+funder
+gitub
```

Comment:
`gitub` seems incorrect

Reply:
Removed from the dictionary in b017f9cd412f004079c6703ff3d7c04ac1eb8394